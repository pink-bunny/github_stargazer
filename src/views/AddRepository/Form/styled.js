import styled from 'styled-components/native';

const StyledView = styled.View`
  flex-grow: 1;
  padding: 20px;
`;
const InputWrapper = styled.View`
  margin-bottom: 16px;
`;
const StyledInput = styled.TextInput`
  border-radius: 3px;
  background-color: white;
  border-color: ${({ isError }) => isError ? 'red' : 'grey'};
  border-width: 1px;
  border-style: solid;
  min-height: 42px;
  margin-bottom: 8px;
  padding-left: 8px;
  padding-right: 8px;
`;
const StyledErrorMessage = styled.View`
  display: flex;
  flex-direction: row;
`;
const StyledErrorMessageText = styled.Text`
  color: red;
`;

export {
  StyledView,
  StyledInput,
  StyledErrorMessage,
  InputWrapper,
  StyledErrorMessageText,
};
