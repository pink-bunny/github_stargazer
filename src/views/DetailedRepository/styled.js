import styled from 'styled-components/native';

const ContentWrap = styled.ScrollView`
  padding: 20px 10px;
`;

const Title = styled.Text`
  font-weight: 700;
  margin-bottom: 16;
  font-size: 48;
`;

const TextWrap = styled.View`
  margin-bottom: 48;
`;

const Text = styled.Text`
  margin-bottom: 12;
  font-size: 24;
`;

export {
  Title, Text, TextWrap, ContentWrap,
};
