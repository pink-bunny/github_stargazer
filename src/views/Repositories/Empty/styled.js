import styled from 'styled-components/native';

const EmptyWrap = styled.View`
  flex-grow: 1;
  align-items: center;
  justify-content: center;
`;

export { EmptyWrap };
