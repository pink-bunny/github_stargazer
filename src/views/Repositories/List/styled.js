import styled from 'styled-components/native';

const StyledItem = styled.View`
  flex-grow: 1;
  padding: 20px;
  border-bottom-width: 1px;
  border-bottom-color: #c5c5c5;
  flex-direction: row;
  justify-content: space-between;
  background-color: white;
`;

export {
  StyledItem,
};
