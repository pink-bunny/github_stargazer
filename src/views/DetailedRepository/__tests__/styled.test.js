import * as styled from '../styled';

it('Styles to match snapshot', () => {
  expect(styled).toMatchSnapshot();
});

