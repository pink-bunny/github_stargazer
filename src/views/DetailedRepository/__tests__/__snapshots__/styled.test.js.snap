// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles to match snapshot 1`] = `
Object {
  "ContentWrap": Object {
    "$$typeof": Symbol(react.forward_ref),
    "attrs": Array [],
    "displayName": "Styled(View)",
    "inlineStyle": InlineStyle {
      "rules": Array [
        "
  padding: 20px 10px;
",
      ],
    },
    "render": [Function],
    "shouldForwardProp": undefined,
    "styledComponentId": "StyledNativeComponent",
    "target": [Function],
    "withComponent": [Function],
  },
  "Text": Object {
    "$$typeof": Symbol(react.forward_ref),
    "attrs": Array [],
    "displayName": "Styled(Text)",
    "inlineStyle": InlineStyle {
      "rules": Array [
        "
  margin-bottom: 4px;
  font-size: 14px;
",
      ],
    },
    "render": [Function],
    "shouldForwardProp": undefined,
    "styledComponentId": "StyledNativeComponent",
    "target": [Function],
    "withComponent": [Function],
  },
  "TextWrap": Object {
    "$$typeof": Symbol(react.forward_ref),
    "attrs": Array [],
    "displayName": "Styled(View)",
    "inlineStyle": InlineStyle {
      "rules": Array [
        "
  margin-bottom: 12px;
",
      ],
    },
    "render": [Function],
    "shouldForwardProp": undefined,
    "styledComponentId": "StyledNativeComponent",
    "target": [Function],
    "withComponent": [Function],
  },
  "Title": Object {
    "$$typeof": Symbol(react.forward_ref),
    "attrs": Array [],
    "displayName": "Styled(Text)",
    "inlineStyle": InlineStyle {
      "rules": Array [
        "
  font-weight: 700;
  margin-bottom: 16px;
  font-size: 16px;
",
      ],
    },
    "render": [Function],
    "shouldForwardProp": undefined,
    "styledComponentId": "StyledNativeComponent",
    "target": [Function],
    "withComponent": [Function],
  },
}
`;
